cli/pack/app_dir.go

```diff
@@ -24,9 +24,11 @@ const (
 	versionFileName    = "VERSION"
 	versionLuaFileName = "VERSION.lua"
 
-	cacheParamsFileName     = "pack-cache.yml"
+	cacheParamsFileName     = "pack-cache-config.yml"
 	maxCachedProjects       = 5
 	cntFirstSymbolsFromHash = 10
+
+	cacheParamsErrorMsg = "You should set one of `always-true`, `key` and `key-path` for path %s"
```

Comment:
Yes, you can use this error message for all cases, but change it:

"You should set one of" -> "You should set one and only one of"

Reply:
Maybe CLI can be a little more polite and say "Please, specify...". Nobody likes when he should do smth.